src/vellum/workflows/nodes/utils.py

```diff
@@ -195,7 +196,34 @@ def _clean_output_type(output_type: Any) -> Any:
     return output_type
 
 
+def _get_default_value(output_type: Any) -> Any:
+    origin = get_origin(output_type)
+    args = get_args(output_type)
+
+    if output_type is int or output_type is float:
+        return 0.0
+    elif output_type is str:
+        return ""
+    elif origin is list:
+        return []
+    elif output_type is FunctionCall:
+        return {}
+    elif origin is Union:
+        # Recursively find first type with a non-None default
+        for inner_type in args:
```

Comment:
[nit] I think we just always use the first arg, since `None` is a valid default return value